app/code/community/Drip/Connect/Model/Configuration.php

```diff
@@ -66,6 +66,11 @@ public function getStoreId()
         return $this->storeId;
     }
 
+    public function getWebsiteId()
```

Comment:
I did a poor job with the rest of these, but could we start a pattern of docblocking new methods in here?

```php
/**
 * The website ID attached to the current store view.
 *
 * @returns int
 */
```